apps/pie-microsite/package.json

```diff
@@ -11,21 +11,26 @@
     "build": "DEPLOYMENT_ENVIRONMENT='production' eleventy",
     "build:dev": "DEPLOYMENT_ENVIRONMENT='development' eleventy",
     "clean:dist": "run -T rimraf dist",
+    "test": "run -T jest",
```

Comment:
Super nit – can we keep scripts ordered alphabetically? Just makes finding them a bit easier in the config